widgetsnbextension/package.json

```diff
@@ -33,5 +33,8 @@
     "rimraf": "^2.5.1",
     "style-loader": "^0.13.0",
     "webpack": "^1.12.12"
+  },
+  "optionalDependencies": {
+    "jupyter-js-widgets": "2.0.0-dev"
```

Comment:
How about we let it be a dependency and use `file: ` all the time except on releases?